### PR TITLE
Yd/adds streaming app

### PR DESCRIPTION
**Pull request by YauheniDraichykau**

### CODE REVIEW

File: frontend/apps/w3bstreaming/.env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+REACT_APP_SIGNALING_SERVER=ws://127.0.0.1:3001
+REACT_APP_STREAMING_PROGRAM_ADDRESS=0x3eb6f1f66560ee83603c8349c7959912266e7acded8b537cdbe4412937cad62a
+REACT_APP_STREAMING_NODE=wss://testnet.vara-network.io
+REACT_APP_STREAMING_IPFS_GATEWAY=
+REACT_APP_STREAMING_IPFS_NODE=
```
Comment (nikitayutanov) [inline frontend/apps/w3bstreaming/.env.example:5]:
```suggestion
REACT_APP_CONTRACT_ADDRESS=0x3eb6f1f66560ee83603c8349c7959912266e7acded8b537cdbe4412937cad62a
REACT_APP_NODE_ADDRESS=wss://testnet.vara-network.io
REACT_APP_IPFS_GATEWAY_ADDRESS=
REACT_APP_IPFS_ADDRESS=
```
please stick to default naming in order for logPublicEnvs and errorTracking utils to work properly without any additional arguments.

File: frontend/apps/w3bstreaming/.prettierrc
Comment (nikitayutanov) [inline frontend/apps/w3bstreaming/.prettierrc:1]: we have root prettierrc

File: frontend/apps/w3bstreaming/.gitignore
Comment (nikitayutanov) [inline frontend/apps/w3bstreaming/.gitignore:1]: we have root gitignore

File: frontend/apps/w3bstreaming/package.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@craco/craco": "^7.1.0",
+    "@types/lodash.merge": "^4.6.7",
+    "@types/node": "18.17.15",
+    "@types/react": "18.2.24",
+    "@types/react-datepicker": "^4.11.2",
+    "@types/react-dom": "18.2.8",
+    "@types/react-gtm-module": "2.0.1",
```
Comment (nikitayutanov) [inline frontend/apps/w3bstreaming/package.json:12]: please use error tracking and analytics utils from shared packages, remove direct dependencies tied to it

File: frontend/apps/w3bstreaming/package.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    "@typescript-eslint/parser": "6.7.3",
+    "eslint-config-airbnb": "19.0.4",
```
Comment (nikitayutanov) [inline frontend/apps/w3bstreaming/package.json:14]: also provide strict versions for packages

**Review by nikitayutanov — CHANGES_REQUESTED** [review]

**Comment by YauheniDraichykau** [comment]
> Everything is good, but I don't see `initErrorTracking()`, `logPublicEnvs()` and `initAnalytics()` calls from `index.tsx`?

Fixed

**Review by ereburg — APPROVED** [review]

**Review by nikitayutanov — APPROVED** [review]

**Pull request merged**